.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,32 @@
+name: Pull Request
+on: pull_request
+
+jobs:
+  build_windows:
+    name: Windows
+    runs-on: windows-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: subosito/flutter-action@v1
+        with:
+          channel: "beta"
+
+      - run: flutter config --enable-windows-desktop
+      - run: flutter pub get
+      - run: flutter build windows --verbose
```

Comment:
Add a github artifact for build files so that anyone can test what really does a PR change. This can be zip executable for windows and .tar.gz for linux.

Reply:
I just added executable zip to upload as artifact. Easy for testing without any further build instructions or requirements. Since, we'll already have all the shared libraries installed on our machines.